tests/playbooks/known_issues.yaml

```diff
@@ -18,3 +18,8 @@
   meta: end_play
   when: "skip_{{ current_playbook }} == true"
 
+# This is a workaround for https://github.com/kubevirt/ansible-kubevirt-modules/issues/240
+- name: "Travis/minikube: give cluster time to recover from its instability"
```

Comment:
Why is this only in known_issues?

Reply:
I'll file an upstream (minikube) bug for this, but need to take care of other stuff first.